projects/sdr_paper/pytorch_experiments/experiments.cfg

```diff
@@ -31,6 +31,12 @@ datadir = "data"
 
 optimizer = SGD
 
+; Learning Rate Scheduler. See "torch.optim.lr_scheduler" for valid class names
+lr_scheduler = "StepLR"
+
+; Configure lr_scheduler class constructor using kwargs style dictionary
+lr_scheduler_params = "{'step_size': 1, 'gamma':%(learning_rate_factor)s}"
```

Comment:
I think we should keep because it is easier to create experiments list on the learning_rate_factor.  See `experiments_cnn.cfg` experiment [cnn6] and [cnn10]

Reply:
Ah got it. 👍